ui/package.json

```diff
@@ -22,7 +22,7 @@
     "bootstrap": "^5.2.1",
     "camelcase": "^6.3.0",
     "css-loader": "6.7.1",
-    "dotenv": "16.0.3",
+    "dotenv": "16.0.2",
```

Comment:
Downgrade?

Reply:
Nope. Forgot to updated my forked branch :)